.gitignore

```diff
@@ -1,3 +1,4 @@
 /.DS_Store
 /site
+docs/biuld/
```

Comment:
```suggestion
docs/build/
```

Reply:
I think because of this typo you committed a lot of files that shouldn't have been committed.